qa/verify-version-constants/build.gradle

```diff
@@ -18,6 +18,12 @@ BuildParams.bwcVersions.withIndexCompatible { bwcVersion, baseName ->
   def baseCluster = testClusters.register(baseName) {
       version = bwcVersion.toString()
       setting 'xpack.security.enabled', 'true'
+      if (BuildParams.inFipsJvm && bwcVersion.before('8.0.0')) {
```

Comment:
This is the only test that has been failing with this particular issue. As such, I'm fixing only the build for this particular test (since it's been bugging people for some time now).

I will follow this up with another PR that addresses with more generally, probably here: https://github.com/elastic/elasticsearch/blob/master/build-tools-internal/src/main/groovy/elasticsearch.fips.gradle#L68